task_1.py

```diff
@@ -0,0 +1,45 @@
+"""
+Задание 1: Доработаем задачи 5-6. Создайте класс-фабрику. Класс принимает тип животного (название одного из созданных классов)
+и параметры для этого типа. Внутри класса создайте экземпляр на основе переданного типа и верните его из класса-фабрики.
+"""   
+
+class Fabrica():
+    def __init__(self, cls, param) -> None:
```

Comment:
Наверное `param` лучше сделать переменным позиционным
```suggestion
    def __init__(self, cls, *param) -> None:
```